cookbook/live-preview.rst

```diff
@@ -0,0 +1,171 @@
+
+    {% block content %}
+        <div id="content" vocab="http://schema.org/" typeof="Content">
+            <h1 property="title">{{ content.title }}</h1>
+        </div>
+    {% endblock %}
+
+**master.html.twig**
+
+.. code-block:: twig
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <title>{{ content.title }}
+        </head>
+        <body>
+            {% block content %}{% endblock %}
+        </body>
+    </html>
+
+**page.xml**
+
+.. code-block:: xml
+
+    <?xml version="1.0" ?>
+    <template xmlns="http://schemas.sulu.io/template/template"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">
+
+        <key>page</key>
+
+        <view>ClientWebsiteBundle:templates:default</view>
+        <controller>SuluWebsiteBundle:Default:index</controller>
+        <cacheLifetime>2400</cacheLifetime>
+
+        <properties>
+            <section name="highlight">
+                <properties>
+                    <property name="title" type="text_line" mandatory="true">
+                        <params>
+                            <param name="headline" value="true"/>
+                        </params>
+
+                        <tag name="sulu.rlp.part"/>
+                    </property>
+
+                    <property name="url" type="resource_locator" mandatory="true">
+                        <tag name="sulu.rlp"/>
+                    </property>
+                </properties>
+            </section>
+        </properties>
+    </template>
+
+This page consists of a single title and a url. The template is quite basic but
+includes the RDFa attribute `property=title` to tell sulu that the title is
+rendered in this container.
+
+When the title of the page will be updated in the content form the admin will
+send this change to the server where the twig block `content` will be rendered.
+The server response only the part which was changed (indicated by
+`property=title`). Back in the javascript application the preview on the right
+will update the container which is marked with the particular property.
+
+This process ensures a quite good performance (as good as the rendering on the
+server side) and minimize the load of the requests.
+
+Supported RDFa features
+-----------------------
+
+To see a full example of the syntax take a look in the
+`example.html.twig <https://github.com/sulu/sulu-standard/blob/develop/src/Client/Bundle/WebsiteBundle/Resources/themes/default/templates/example.html.twig>`
+file.
+
+Links
+*****
+
+For links you can use the href-attribute which will also be updated if the
+mentioned property was changed.
+
+.. code-block:: twig
+
+    <a href="{{ sulu_content_path(content.link.url) }}" property="link">
+        {{ content.link.title }}
+    </a>
+
+Images
+******
+
+As like the links you can simple use the src-attribute to update an image
```

Comment:
@wachterjohannes I would rewrite this sentence. Maybe: `To update an image you can simply use the src-attribute when the mentioned property has changed`